predict.py

```diff
@@ -10,26 +11,52 @@
     StableDiffusionPipeline,
 )
 
+# Stable Diffusion Defaults
 MODEL_ID = "stabilityai/stable-diffusion-2-1"
 MODEL_CACHE = "diffusers-cache"
+DEFAULT_HEIGHT = 768
+DEFAULT_WIDTH = 768
+DEFAULT_SCHEDULER = "K_EULER"
+DEFAULT_PROMPT = "a photo of an astronaut riding a horse on mars"
+
+# Dreambooth Training Defaults
+USE_DREAMBOOTH_WEIGHTS = os.path.exists("weights")
```

Comment:
Is there a world in which a user might bring their own weights dir that _isn't_ DreamBooth? If so, we might want to name this something more generic than `USE_DREAMBOOTH_WEIGHTS`, like `USE_LOCAL_WEIGHTS_DIR` or similar.

Reply:
I can go either way -- It isn't dreambooth specific... but the default stable-diffusion weights are also "local":

            cache_dir=MODEL_CACHE,
            local_files_only=True,

That was my reasoning - given that it isn't an environmental variable / used anywhere but this file - I am ok with either name.  

Another option could be to allow defaults to come in via environmental variables that get set during build time (I'm not sure `cog` supports this, but docker does)

Then all the settings DEFAULT_PROMPT/WIDTH/HEIGHT/SCHEDULER/... could be set at build time - for instance by the dreambooth builder api

(this is probably a better idea - but I propose merging this now -- or deciding to not merge and make the changes to `dreambooth-template` to unify with this repo ... instead of spending too long iterating on this non-user facing implementation detail)